packages/ui/exports/types/form-core.ts

```diff
@@ -1,5 +1,6 @@
 export { ChoiceGroupHost } from '../../components/form-core/types/choice-group/ChoiceGroupMixinTypes.js';
 export { ChoiceInputHost } from '../../components/form-core/types/choice-group/ChoiceInputMixinTypes.js';
+export { CustomChoiceGroupHost } from '../../components/form-core/types/choice-group/CustomChoiceGroupMixinTypes.js';
```

Comment:
```suggestion
```
For now, we don't want to expose the mixin yet (as it probably will be merged into ChoiceGroupMixin)